lib/python/qmk/cli/doctor.py

```diff
@@ -80,13 +83,13 @@ def check_avr_gcc_version():
 
         parsed_version = parse_gcc_version(version_number)
         if parsed_version['major'] > 8:
-            cli.log.error('We do not recommend avr-gcc newer than 8. Downgrading to 8.x is recommended.')
-            return False
+            cli.log.error('{fg_yellow}We do not recommend avr-gcc newer than 8. Downgrading to 8.x is recommended.')
```

Comment:
Currently doctor does not report the version in this codepath. If we display this warning we should do that too.

Reply:
Nice catch.